web/wp-content/mu-plugins/pantheon/pantheon-login-form-mods.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Modify the WordPress login form for Pantheon
+ */
+
+ /**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
+  */
+$show_return_to_pantheon_button = apply_filters( 'show_return_to_pantheon_button', ( 
+    ( ! defined('RETURN_TO_PANTHEON_BUTTON') || RETURN_TO_PANTHEON_BUTTON ) &&
+    (
+        false !== stripos( get_site_url(), 'pantheonsite.io') ||
+        ( isset( $_SERVER['HTTP_HOST'] ) && false !== stripos( $_SERVER['HTTP_HOST'], 'pantheonsite.io') )
+     )
+) );
+
+if( $show_return_to_pantheon_button ){
+
+    /**
+     * Enqueue Pantheon login styles
+     *
+     * @return void
+     */
+    function Pantheon_Enqueue_Login_style()
+    {
+        wp_enqueue_style('pantheon-login-mods', plugin_dir_url(__FILE__) . 'assets/css/return-to-pantheon-button.css', false); 
+    }
+
+    add_action('login_enqueue_scripts', 'Pantheon_Enqueue_Login_style', 10);
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).